rasa_nlu/server.py

```diff
@@ -58,6 +59,8 @@ def decorated(*args, **kwargs):
 
 def create_app(config, component_builder=None):
     rasa_nlu_app = Flask(__name__)
+    if 'cors' in config and config['cors'] == True:
+        CORS(rasa_nlu_app)
```

Comment:
Not sure about how to achieve it, but would it be possible to use `CORS(app, resources={r"/api/*": {"origins": config['cors']}})`?  
With `cors` being a configuration setting accepting either a string or a list of URIs (strings).